torchvision/datasets/mnist.py

```diff
@@ -7,6 +7,8 @@
 import torch
 import codecs
 import string
+import gzip
+import lzma
```

Comment:
Is it useful to import lzma ?

Reply:
Not sure what you are asking here: why did I remove the lazy import of `lzma` (1.) or why do we import `lzma` at all (2.)?

1. I needed to remove the lazy import of `gzip` since I needed it for the type annotation. Given that both `gzip` and `lzma` are part of the standard library I see no point in lazily importing them for a micro optimization. That being said, I have no strong opinion here. I can revert the changes to `lzma` if you like.
2. Given that `lzma` is used in `open_maybe_compressed_file` we probably need it. If this branch is not used during operation, I think we should remove it in a separate PR.

Reply:
@pmeier sorry, I didn't remarked that lzma was imported lazily below. Thanks for the explanation !

Reply:
For reference, lzma was introduced to the standard library in Python 3.3, and we used to support Python 2 as well. As we dropped support for Python 2, importing lzma in the main namespace is ok now.